### PR TITLE
Add ListBox control

### PR DESCRIPTION
**Pull request by cema-sp**
Adding ListBoxControl component with items as props.

### CODE REVIEW

**Comment by gribnoysup** [comment]
Hi @cema-sp, thank you for your PR!

What do you think about making ListBoxItem a separate component and not just the prop for ListBoxControl? This way you could use it like this:

```jsx
<ListBox>
  <ListBoxItem onClick={() => {}} data={{ content: 'Dzerzhinsky' }}/>
  // ...
```

This looks more like other APIs for ymaps objects in this library

Maybe you already tried it and there are some pitfalls in this approach? Let me know!

**Comment by cema-sp** [comment]
Hi @gribnoysup !

I've already tried to pass `ListBoxItem`s as children of `ListBoxControl` to `props.items` of `Control` component but `ymaps` lib [requires](https://tech.yandex.ru/maps/doc/jsapi/2.1/ref/reference/control.ListBox-docpage/#param-parameters.items) items to be instances of `ymaps.control.ListBoxItem`.

We could extract props from `ListBoxItem` and create `ymaps.control.ListBoxItem` based on them if it's worth it.

**Comment by gribnoysup** [comment]
@cema-sp I was thinking more like allowing `<Control>` component to render its children and them `<ListBoxItem>` would add itself to the Control collection

I'll start a review and will point out in more detail what I mean

File: src/Control.js [hunk 1]
```diff
@@ -22,32 +22,66 @@ export class Control extends React.Component {
...
+      events,
+      items,
+      instanceRef,
+    } = separateEvents(this.props);
     const { ymaps, parent } = this.context;
 
     const instance = new ymaps.control[type]({ data, options, state });
 
     Object.keys(events).forEach(key => addEvent(events[key], key, instance));
     parent.controls.add(instance);
```
Comment (gribnoysup) [inline src/Control.js]: ... and then in this line you can check if `parent` have `add` method (like in [`<GeoObject>` component](https://github.com/gribnoysup/react-yandex-maps/blob/master/src/GeoObject.js#L53)).

If it does, just add it straight to `parent`

**Comment by gribnoysup** [comment]
Closing because of no progress on the pull request

**Pull request closed without merging**